README.md

```diff
@@ -75,6 +75,20 @@ The following customizations are available in the `LinstorCluster` resource:
   spec:
     linstorPassphraseSecret: my-linstor-passphrase
   ```
+* A reference to a TLS secret, containing the secret key and certificate use in mutual authentication of  the satellites.
```

Comment:
We should document the implicit naming of secrets and show how to configure TLS without `cert-manager`.

Reply:
Good point. I'm wondering if this would be a good time to think about proper splitting for our documentation (see also https://diataxis.fr/)

The README is already quite long, with lots of information that's probably not needed, while actual reference documentation is not available.

Reply:
Sounds good. Do you have a plan already for implementing that?

We can merge this PR and then work on the docs.

Reply:
Having it written down will surely help, but now I think we need ChatGPT to generate all that text :sweat_smile: https://github.com/piraeusdatastore/piraeus-operator/issues/368